src/diem/testing/suites/conftest.py

```diff
@@ -29,7 +33,7 @@ def diem_client() -> jsonrpc.Client:
 
 @pytest.fixture(scope="package")
 def stub_config() -> AppConfig:
-    return AppConfig(name="stub-wallet", enable_debug_api=True)
+    return AppConfig(name="stub-wallet", enable_debug_api=True, server_conf=ServerConfig(**dmw_stub_server()))
```

Comment:
this is the only place uses this dmw_stub_server, I think it is simpler just inline the change, which then will remove `**`

Reply:
even if I inline the change, will still need `**`- the line would then look like 

```
@pytest.fixture(scope="package")
def stub_config() -> AppConfig:
    return AppConfig(name="stub-wallet", enable_debug_api=True, server_conf=ServerConfig(**{k: v for k, v in {"port": dmw_stub_bind_port(), "base_url": dmw_stub_diem_account_base_url()}.items() if v}))
```

the reason being that dataclass' `field` will accept a `None` and then not set the default